<ui version="4.0" >
 <class>ScalarTab</class>
 <widget class="QWidget" name="ScalarTab" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>343</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QRadioButton" name="_readFromSource" >
     <property name="text" >
      <string>Read from data &amp;source</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QRadioButton" name="_readFromRVector" >
     <property name="whatsThis" >
      <string>Generate a new vector with the properties given here as the independent variable.</string>
     </property>
     <property name="text" >
      <string>Read from data &amp;vector</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QRadioButton" name="_generateX" >
     <property name="whatsThis" >
      <string>Generate a new vector with the properties given here as the independent variable.</string>
     </property>
     <property name="text" >
      <string>&amp;Generate</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="_dataScalarGroup" >
     <property name="title" >
      <string>Read from data source</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>File name:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="Kst::DataSourceSelector" native="1" name="_fileName" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Field:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_field</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="Kst::ComboBox" name="_field" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>&amp;Field:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_fieldRV</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="Kst::ComboBox" name="_fieldRV" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" name="_3" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>21</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="_connect" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>&amp;Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_configure" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Co&amp;nfigure...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2" >
       <layout class="QHBoxLayout" name="_rVectorFrames" >
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>F&amp;rame:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_start</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_start" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="toolTip" >
           <string>Input a number, or the format [[[[YYYY:]MM:]DD:]HH:]MM[.SS] for dates.</string>
          </property>
          <property name="whatsThis" >
           <string>The index of the first data read.  For ASCII files, this is the data row number in the file.  For frame based files, this is the first frame.</string>
          </property>
          <property name="text" >
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Or</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="_countFromEnd" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="whatsThis" >
           <string>Select this to read data from the end of the file.  This is useful for scrolling displays when the file is being appended by another application.</string>
          </property>
          <property name="text" >
           <string>&amp;Last Frame</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QGroupBox" name="_generatedScalarGroup" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Generate</string>
     </property>
     <layout class="QGridLayout" name="_4" >
      <property name="topMargin" >
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2" >
       <layout class="QHBoxLayout" name="_5" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>V&amp;alue: </string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
          <property name="buddy" >
           <cstring>_scalarValue</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_scalarValue" >
          <property name="whatsThis" >
           <string>The maximum value of the independent variable.</string>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>Kst::DataSourceSelector</class>
   <extends>QWidget</extends>
   <header>datasourceselector.h</header>
  </customwidget>
  <customwidget>
   <class>Kst::ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
